features/bootstrap/FeatureContext.php

```diff
@@ -1,20 +0,0 @@
-<?php
```

Comment:
Please can we reinstate this file?

The reason being, on https://wordhat.info/installation/index.html on step 3 and 4, `vendor/behat --init` throws a warning and does not create `/features/bootstrap/featurecontext.php`  unless this file existed